src/config.cpp

```diff
@@ -126,8 +127,7 @@ bool Config::validate() {
 
     // metrics on rest port
     if ((metricsEnabled() || !metricsList().empty()) && !configPath().empty()) {
-        std::cerr << "metrics_enable or metrics_list and config_path cant be used together. Use json config file to enable metrics when using config_path." << std::endl;
```

Comment:
We should add **error** in case someone has CLI metric configuration and config.json metric configuration.
We should also have **error** in case somebody adds metric configuration into config.json at runtime while having CLI metric configuration from the beginning.
We should not pass this information through `startFromFile/loadConfig` calls.